app/views/camp/add_new.php

```diff
@@ -0,0 +1,38 @@
+<h2>Add a new camp</h1>
+<?php
+
+if ($added !== null) {
+    if ($added == true) {
+        echo "Added successfully";
+    }
+    else {
+        echo "Submission failed";
+    }
+    echo "<br><br>";
+}
+?>
+<form action="" method="POST">
+    
+        <legend>Basic details about the Camp</legend>
+        <label for="name">Name: </label><input id="name" name="name" required /><br>
+        <label for="organisation_id">Organisation: </label><input id="organisation_id" name="organisation_id" required /><br/>
+          <!--  <select id="organisation" name="camp[organisation]" required>
+                <option value="" disabled selected>Select Organisation</option>
+                <option value="F">Female</option>
+                <option value="M">Male</option>
+                <option value="O">Other</option>
+            </select><br>-->
+        <label for="camp_head">Camp head: <input id="camp_head" name="camp_head" required /></label><br/>
```

Comment:
That one has bigger problems.. :/ So I thought we'll fix this and merge it. Anyway, your choice.